packages/server/src/routes/api/resource/draft/approve/[_id].ts

```diff
@@ -67,6 +68,24 @@ export async function post(req, res, next) {
     );
     await DraftResource.deleteByRecordId(draftToApprove._id);
 
+    try {
+      const updatedResourceObject = await resourceDocumentToResource(
+        updatedResource
+      );
+      await algoliaIndex.saveObject({
```

Comment:
We'll want an extra conditional to handle when a draft soft deletes the resource instead.

Reply:
Good point, I didn't consider that deletion also happens from this endpoint. I'll update with that asap